sdcm/cluster.py

```diff
@@ -1343,7 +1342,7 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
             p = re.compile('[# ]*cluster_name:.*')
             scylla_yaml_contents = p.sub('cluster_name: {0}'.format(cluster_name),
                                          scylla_yaml_contents)
-        if hinted_handoff_disabled:  # disable hinted handoff (it is enabled by default in Scylla)
+        if hinted_handoff == 'disabled':  # disable hinted handoff (it is enabled by default in Scylla). Expected values: "enabled"/"disabled"
             p = re.compile('[# ]*hinted_handoff_enabled:.*')
```

Comment:
need to fix here the issue of double parameter:

after running the code you get since in the original file there are two examples of using this parameter
```
# May either be "true" or "false" to enable globally, or contain a list
# of data centers to enable per-datacenter.
hinted_handoff_enabled: false
hinted_handoff_enabled: false

```

Reply:
Ok

Reply:
Fixed